File: Controle Arduino Desktop/Controle Arduino Desktop/Form1.vb
Public Class Form1
    Private Sub btnLigLedAmarelo_Click(sender As Object, e As EventArgs) Handles btnLigLedAmarelo.Click
        Dim ledController As New LedController()

        Dim led As Led = LedController.LigarLedAmarelo()

        entLed.Text = led.cor
        entStaLed.Text = led.statusLed

    End Sub

    Private Sub btnDesLedAmarelo_Click(sender As Object, e As EventArgs) Handles btnDesLedAmarelo.Click
        Dim ledController As New LedController()

        Dim led As Led = LedController.DesligarLedAmarelo()

        entLed.Text = led.cor
        entStaLed.Text = led.statusLed

    End Sub

    Private Sub btnStaLedAmarelo_Click(sender As Object, e As EventArgs) Handles btnStaLedAmarelo.Click
        Dim ledController As New LedController()

        Dim led As Led = LedController.StatusLedAmarelo()

        entLed.Text = led.cor
        entStaLed.Text = led.statusLed

    End Sub

    Private Sub btnLigLedVerde_Click(sender As Object, e As EventArgs) Handles btnLigLedVerde.Click
        Dim ledController As New LedController()

        Dim led As Led = LedController.LigarLedVerde()

        entLed.Text = led.cor
        entStaLed.Text = led.statusLed

    End Sub

    Private Sub btnDesLedVerde_Click(sender As Object, e As EventArgs) Handles btnDesLedVerde.Click
        Dim ledController As New LedController()

        Dim led As Led = LedController.DesligarLedVerde()

        entLed.Text = led.cor
        entStaLed.Text = led.statusLed

    End Sub

    Private Sub btnStaLedVerde_Click(sender As Object, e As EventArgs) Handles btnStaLedVerde.Click
        Dim ledController As New LedController()

        Dim led As Led = LedController.StatusLedVerde()

        entLed.Text = led.cor
        entStaLed.Text = led.statusLed

    End Sub

    Private Sub btnLigSomAlarme_Click(sender As Object, e As EventArgs) Handles btnLigSomAlarme.Click
        Dim BuzzerController As New BuzzerController()

        Dim buzzer As Buzzer = BuzzerController.TocarBuzzerAlarme()

        entSom.Text = buzzer.tipoSom
        entStaSom.Text = buzzer.status

    End Sub

    Private Sub btnPauSomAlarme_Click(sender As Object, e As EventArgs) Handles btnPauSomAlarme.Click
        Dim BuzzerController As New BuzzerController()

        Dim buzzer As Buzzer = BuzzerController.PausarBuzzerAlarme()

        entSom.Text = buzzer.tipoSom
        entStaSom.Text = buzzer.status

    End Sub

    Private Sub btnDesSomAlarme_Click(sender As Object, e As EventArgs) Handles btnDesSomAlarme.Click
        Dim BuzzerController As New BuzzerController()

        Dim buzzer As Buzzer = BuzzerController.DesligarBuzzerAlarme()

        entSom.Text = buzzer.tipoSom
        entStaSom.Text = buzzer.status

    End Sub

    Private Sub btnStatusSom_Click(sender As Object, e As EventArgs) Handles btnStatusSom.Click
        Dim BuzzerController As New BuzzerController()

        Dim buzzer As Buzzer = BuzzerController.StatusBuzzerAlarme()

        entSom.Text = buzzer.tipoSom
        entStaSom.Text = buzzer.status

    End Sub

    Private Sub btnLigSomAmbulancia_Click(sender As Object, e As EventArgs) Handles btnLigSomAmbulancia.Click
        Dim BuzzerController As New BuzzerController()

        Dim buzzer As Buzzer = BuzzerController.TocarBuzzerAmbulancia()

        entSom.Text = buzzer.tipoSom
        entStaSom.Text = buzzer.status


    End Sub

    Private Sub btnPauSomAmbulancia_Click(sender As Object, e As EventArgs) Handles btnPauSomAmbulancia.Click
        Dim BuzzerController As New BuzzerController()

        Dim buzzer As Buzzer = BuzzerController.PausarBuzzerAmbulancia()

        entSom.Text = buzzer.tipoSom
        entStaSom.Text = buzzer.status

    End Sub

    Private Sub btnDesSomAmbulancia_Click(sender As Object, e As EventArgs) Handles btnDesSomAmbulancia.Click
        Dim BuzzerController As New BuzzerController()

        Dim buzzer As Buzzer = BuzzerController.DesligarBuzzerAmbulancia()

        entSom.Text = buzzer.tipoSom
        entStaSom.Text = buzzer.status

    End Sub
End Class

File: Controle Arduino Desktop C-SHARP/Controle Arduino Desktop C-SHARP/Form1.cs
using Controle_Arduino_Desktop_C_SHARP.Controller;
using Controle_Arduino_Desktop_C_SHARP.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Controle_Arduino_Desktop_C_SHARP
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnLigLedVerde_Click(object sender, EventArgs e)
        {
            var ledController = new LedController();

            var led = (Led) ledController.LigarLedVerde();

            entLed.Text = led.cor;
            entStaLed.Text = led.statusLed;
        }

        private void btnDesLedVerde_Click(object sender, EventArgs e) {
            {
                var ledController = new LedController();

                var led = (Led)ledController.DesligarLedVerde();

                entLed.Text = led.cor;
                entStaLed.Text = led.statusLed;

            }


        }

        private void btnStaLedVerde_Click(object sender, EventArgs e)  
        {
            var ledController = new LedController();

            var led = (Led) ledController.StatusLedVerde();

            entLed.Text = led.cor;
            entStaLed.Text = led.statusLed;
        }

        private void btnLigLedAmarelo_Click(object sender, EventArgs e)
        {
            var ledController = new LedController();

            var led = (Led) ledController.LigarLedAmarelo();

            entLed.Text = led.cor;
            entStaLed.Text = led.statusLed;
        }

        private void btnDesLedAmarelo_Click(object sender, EventArgs e)
        {

            var ledController = new LedController();

            var led = (Led)ledController.DesligarLedAmarelo();

            entLed.Text = led.cor;
            entStaLed.Text = led.statusLed;
        }

        private void btnStaLedAmarelo_Click(object sender, EventArgs e)
        {
            var ledController = new LedController();

            var led = (Led) ledController.StatusLedAmarelo();

            entLed.Text = led.cor;
            entStaLed.Text = led.statusLed;

        }

        private void btnLigAlarme_Click(object sender, EventArgs e)
        {
            var buzzerController = new BuzzerController();

            var buzzer = (Buzzer) buzzerController.TocarBuzzerAlarme();

            entSom.Text = buzzer.tipoSom;
            entStaSom.Text = buzzer.status;

        }

        private void btnPauAlarme_Click(object sender, EventArgs e)
        {
            var buzzerController = new BuzzerController();

            var buzzer = (Buzzer)buzzerController.PausarBuzzerAlarme();

            entSom.Text = buzzer.tipoSom;
            entStaSom.Text = buzzer.status;
        }

        private void btnDesAlarme_Click(object sender, EventArgs e)
        {
            var buzzerController = new BuzzerController();

            var buzzer = (Buzzer) buzzerController.DesligarBuzzerAlarme();

            entSom.Text = buzzer.tipoSom;
            entStaSom.Text = buzzer.status;
        }

        private void btnStaSom_Click(object sender, EventArgs e)
        {
            var BuzzerController = new BuzzerController();

            var buzzer = (Buzzer)BuzzerController.StatusBuzzerAlarme();

            entSom.Text = buzzer.tipoSom;
            entStaSom.Text = buzzer.status;
        }

        private void btnLigAmbulancia_Click(object sender, EventArgs e)
        {
            var BuzzerController = new BuzzerController();

            var buzzer = (Buzzer) BuzzerController.TocarBuzzerAmbulancia();

            entSom.Text = buzzer.tipoSom;
            entStaSom.Text = buzzer.status;
        }

        private void btnPauAmbulancia_Click(object sender, EventArgs e)
        {
            var BuzzerController = new BuzzerController();

            var buzzer = (Buzzer) BuzzerController.PausarBuzzerAmbulancia();

            entSom.Text = buzzer.tipoSom;
            entStaSom.Text = buzzer.status;
        }

        private void btnDesAmbulancia_Click(object sender, EventArgs e)
        {
            var buzzerController = new BuzzerController();

            var buzzer = (Buzzer) buzzerController.DesligarBuzzerAmbulancia();

            entSom.Text = buzzer.tipoSom;
            entStaSom.Text = buzzer.status;
        }
    }
}
